configfiles.rst

```diff
@@ -241,16 +241,30 @@ allow containers that are located within the specified path prefix.
 
 ``allow container ${type}``: This option allows admins to limit the
 types of image formats that can be leveraged by users with
-{Singularity}. Formats include ``squashfs`` which is used by SIF and
-v2.x Singularity images, ``extfs`` which is used for writable overlays
-and some legacy Singularity images, ``dir`` which is used by sandbox
-images and ``encrypted`` which is only used by SIF images to encrypt
-filesystem contents.
+{Singularity}. 
+
+- ``allow container sif`` permits / denies execution of unencrypted SIF
+  containers.
+
+- ``allow container encrypted`` permits / denies execution of SIF containers with
+  an encrypted root filesystem.
+
+- ``allow container squashfs`` permits / denies execution of bare SquashFS image
+  files. E.g. Singularity 2.x images.
+
+- ``allow container extfs`` permits / denies execution of bare EXT image files.
+
+- ``allow container dir`` permits / denies execution of sandbox directory
+  containers.
 
 .. note::
 
    These limitations do not apply to the root user.
 
+   This behavior differes from {Singularity} versions before 3.9, where the
```

Comment:
differes is a typo but it doesn't matter because we will be removing text referring to older versions.